modules/sharedIdSystem.js

```diff
@@ -10,9 +10,13 @@ import {ajax} from '../src/ajax.js';
 import {submodule} from '../src/hook.js';
 
 const MODULE_NAME = 'sharedId';
-const ID_SVC = 'https://id.sharedid.org/id';
+const SHAREDID_URL = 'https://id.sharedid.org';
+const ID_SVC = SHAREDID_URL + '/id';
+const SYNC_SVC = SHAREDID_URL + '/iframe/optout';
```

Comment:
I get a 404 when calling https://id.sharedid.org/iframe/optout ... is this ready to go?

Reply:
never mind...I missed this comment, sorry! https://github.com/prebid/Prebid.js/pull/6068#issuecomment-736902048